test/unit/utils/validateFilepath.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, afterEach, expect, test, jest } from '@jest/globals';
```

Comment:
Missing `beforeEach` import

Reply:
After `validateFilepath` we do not need the mocking of the `req` and `res` object, so I will resolve the following comment :)

Reply:
Yea can remove these now logic is more simple